.github/workflows/publish-on-release.yml

```
@@ -64,3 +64,20 @@ jobs:
         run: upload-assets --url-path vanilla-framework-version-$(cat css/VANILLA_VERSION).min.css css/build.css
         env:
           UPLOAD_ASSETS_API_TOKEN: ${{secrets.UPLOAD_ASSETS_API_TOKEN}}
+  
+  notify-figma:
+    name: Notify Figma library maintainers
+    runs-on: ubuntu-latest
```

It should only run if publish to npm was successful.

```suggestion
    needs: publish-npm
    runs-on: ubuntu-latest
```